### PR TITLE
fix(storage-browser): fix task table progress icon classname

### PR DESCRIPTION
**Pull request by calebpollman**
#### Description of changes
Update "progress" text data cell icon classname to include "amplify-" prefix

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] `yarn test` passes and tests are updated/added
- [ ] PR title and commit messages follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax
- [ ] _If this change should result in a version bump_, [changeset added](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) (This can be done after creating the PR.) This does not apply to changes made to `docs`, `e2e`, `examples`, or other private packages.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-storage/src/styles/storage-browser.css [hunk 1]
```diff
@@ -217,7 +217,7 @@
 }
 
 .storage-browser__loading-indicator-icon,
-.storage-browser__table-text-data-cell-icon--action-progress {
+.amplify-storage-browser__table-text-data-cell-icon--action-progress {
```
Comment (jordanvn) [inline packages/react-storage/src/styles/storage-browser.css:220]: Info: aren't the `.amplify-storage-browser__` classes handled in `ui/src/theme/css/component/StorageBrowser/storageBrowser.scss`?

**Review by jordanvn — COMMENTED** [review]
These changes make sense to me and I validated the fix locally. Would just like clarification on styling rule placement before approval

**Review by jordanvn — APPROVED** [review]

File: packages/ui/src/theme/css/component/StorageBrowser/storageBrowser.scss [hunk 2]
```diff
@@ -146,6 +146,15 @@
...
       justify-content: flex-start;
+
+      &-icon {
+        &--action-progress {
+          animation-name: amplify-loader-circular;
+          animation-timing-function: linear;
+          animation-iteration-count: infinite;
+          animation-duration: var(--amplify-components-loader-animation-duration);
+        }
+      }
```
Comment (jordanvn) [inline packages/ui/src/theme/css/component/StorageBrowser/storageBrowser.scss:157]: Info: does this need to be within any selector aside from `-text-data-cell`?

Comment (cshfang) [reply packages/ui/src/theme/css/component/StorageBrowser/storageBrowser.scss:157]: Afaict, this was our only use case for this. I considered lifting it up and using wildcard but I think just keeping it simple is probably right for now

**Pull request merged**